src/gatsby-theme-docz/components/Header/index.js

```diff
@@ -0,0 +1,3 @@
+import React from "react";
+
+export const Header = () => <></>;
```

Comment:
I suspect this is just overriding the header to remove it. A brief comment to that effect will make this clear to future maintainers.

Reply:
This will be changing with:
https://github.com/GetJobber/atlantis/pull/217

But, perhaps some sort of documentation around component shadowing would still be good?

I don't know if this is good enough:
https://www.docz.site/docs/component-shadowing

In retrospect it makes sense but, perhaps also
https://www.gatsbyjs.org/blog/2019-04-29-component-shadowing/

To save some learning for the next person?

Reply:
Perhaps a readme in the src directory of the theme?